mlos_core/mlos_core/tests/spaces/adapters/llamatune_test.py

```diff
@@ -34,11 +35,15 @@ def construct_parameter_space(  # pylint: disable=too-many-arguments
     for idx in range(n_continuous_params):
         input_space.add(CS.UniformFloatHyperparameter(name=f"cont_{idx}", lower=0, upper=64))
     for idx in range(n_quantized_continuous_params):
-        input_space.add(CS.UniformFloatHyperparameter(name=f"cont_{idx}", lower=0, upper=64, q=12.8))
+        param_int = CS.UniformFloatHyperparameter(name=f"cont_{idx}", lower=0, upper=64)
+        monkey_patch_quantization(param_int, 6)
+        input_space.add(param_int)
     for idx in range(n_integer_params):
         input_space.add(CS.UniformIntegerHyperparameter(name=f"int_{idx}", lower=-1, upper=256))
     for idx in range(n_quantized_integer_params):
-        input_space.add(CS.UniformIntegerHyperparameter(name=f"int_{idx}", lower=0, upper=256, q=16))
+        param_float = CS.UniformIntegerHyperparameter(name=f"int_{idx}", lower=0, upper=256)
+        monkey_patch_quantization(param_float, 17)
```

Comment:
I wonder if we should add some sort of internal use of `meta` to parse `q` or `quantization_bins` out of that data and use it to apply the monkey patch on demand rather than encoding it explicitly here.

Thoughts?

Reply:
you mean, have something like `monkey_patch(ConfigSpace: cs)` that goes over all hyperparameters and patches them according to the metadata? I like that! I'll push a PR later once we merge this one in

Reply:
Something like that.  Basically, we shouldn't have to remember to call monkey patch - it should be handled for us internally.  At least that's my high level design thought.